openwisp_users/accounts/templates/account/email/email_confirmation_signup_message.html

```diff
@@ -1,4 +1,4 @@
-{% extends "account/email/base_message.html" %}
+{% extends "openwisp_utils/email_template.html" %}
```

Comment:
I think we should not extend from `openwisp_utils/email_template.html` if we intend to pass this as the body argument to `send_email`